pkg/operator/staticpod/controller/missingstaticpod/missing_static_pod_controller.go

```diff
@@ -0,0 +1,263 @@
+package missingstaticpodcontroller
+
+import (
+	"context"
+	"fmt"
+	"sort"
+	"strconv"
+	"strings"
+	"time"
+
+	operatorv1 "github.com/openshift/api/operator/v1"
```

Comment:
nit: please move it down to the `openshift` group.

Reply:
this is actually the way goimports is sorting the imports. First you have go std libs, then external library and finally local libraries.